tests/framework/installer/ceph_helm_installer.go

```diff
@@ -132,8 +132,9 @@ func (h *CephInstaller) configureRookCephClusterViaHelm(upgrade bool) error {
 				"nginx.ingress.kubernetes.io/rewrite-target": "/ceph-dashboard/$2",
 			},
 			"host": map[string]interface{}{
-				"name": "localhost",
-				"path": "/ceph-dashboard(/|$)(.*)",
+				"name":     "localhost",
```

Comment:
do we also deploy the ingress controller?

Reply:
No, I don't think so